src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java

```diff
@@ -85,6 +85,7 @@
     "var context = typeof arguments[0] === 'string' ? JSON.parse(arguments[0]) : arguments[0];" +
     "context = context || document;" +
     "var options = JSON.parse(arguments[1]);" +
+    "axe.configure({ allowedOrigins: ['<unsafe_all_origins>'] });" +

```

Comment:
This should go after injecting axe. Specifically, after this line: https://github.com/dequelabs/axe-core-maven-html/pull/157/files#diff-bbf844feca71b38a2bac26def392f0fecfd45e3d774662feeff8583352ffb79bR451